examples/mxnet/dis_kvstore/client.py

```diff
@@ -0,0 +1,57 @@
+# This is a simple MXNet client demo shows how to use DGL distributed kvstore.
+# In this demo, we initialize two embeddings on server and push/pull data to/from it.
+import dgl
+import mxnet as mx
+import time
+import argparse
+
+server_namebook, client_namebook = dgl.contrib.ReadNetworkConfigure('config.txt')
+
+def start_client(args):
+    # Initialize client and connect to server
+    client = dgl.contrib.KVClient(
+        client_id=args.id, 
+        server_namebook=server_namebook, 
+        client_addr=client_namebook[args.id])
+
+    client.connect()
+
+    # Initialize data on server
+    client.init_data(name='embed_0', shape=[10, 3], init_type='zero')
+    client.init_data(name='embed_1', shape=[11, 3], init_type='uniform', low=0.0, high=0.0)
+
+    tensor_id = mx.nd.array([0, 1, 2], dtype='int64')
+    tensor_data = mx.nd.array([[0., 0., 0., ], [1., 1., 1.], [2., 2., 2.]])
+
+    # Push data
+    for i in range(5):
```

Comment:
Does this for loop has any meaning?

Reply:
> Does this for loop has any meaning?

No, I just want to test sending data 5 times.